pennylane/beta/queuing/measure.py

```diff
@@ -104,7 +104,7 @@ def eigvals(self):
 
         >>> m = MeasurementProcess(Expectation, obs=qml.PauliX(wires=1))
         >>> U.eigvals
-        >>> array([1, -1])
+        array([1, -1])
```

Comment:
@co9olguy 👀

Reply:
:heart_eyes:

Reply:
Note that `U` also seems to be undefined?